src/DetailsView/components/issues-table.tsx

```diff
@@ -95,6 +95,12 @@ export class IssuesTable extends React.Component<IssuesTableProps> {
         );
     }
 
+    private nullUpdatePersistedDescription(value: string): void {}
+
+    private nullGetExportDescription = () => {
```

Comment:
I think since you're returning empty string here, you should probably refer to this as 'getEmptyExportDescription'.

Additionally, you can simplify it by just doing:

private getEmptyExportDescription= () => '';